meetup/models.py

```diff
@@ -1,3 +1,62 @@
+from uuid import uuid4
+
+from django.contrib.auth.models import User
 from django.db import models
 
-# Create your models here.
+
+class Venue(models.Model):
+    """This model is for the venue where be held"""
+    name = models.CharField(max_length=50)
+    location = models.CharField(max_length=255, null=True)
+    description = models.TextField(null=True)
+    latitude = models.FloatField()
+    longitude = models.FloatField()
+
+
+class ProgramCategory(models.Model):
+    """This model is for the category of program"""
+    name = models.CharField(max_length=100)
+    slug = models.SlugField(max_length=100, unique=True)
+
+
+class Speaker(models.Model):
+    """This model is for that user related by speaker at program"""
+    user = models.ForeignKey(User)
+
+
+class Program(models.Model):
+    """This model is for that lecture or event by speaker(s)"""
+    # There should be difficulty and language information in Program?
+    title = models.CharField(max_length=255)
+    brief = models.TextField(null=True, verbose_name='simple information')
```

Comment:
```python
from django.utils.translation import ugettext as _
verbose_name=_('simple information')
```
Please use gettext to description text translation.

Reply:
On my plan, I will translate whole words later at once.
But I think, you are not agree with me. are you?

Reply:
I agree with you.
I think gettext is still right choice if your plan include i18n.

Reply:
Thank you, I want to use the i18n and L10n on this project and [PyCon Korea Website](https://github.com/pythonkr), for Best Practice :)